_data/sw_libraries.yml

```diff
@@ -80,3 +80,9 @@
           - userhost-in-names
           - batch
           - sasl
+    - name: BlueIRC
+      # ref: https://github.com/MicroDroid/blueirc/blob/development/src/tk/microdroid/blueirc/Handler.java#L59
+      v3.1:
```

Comment:
oh I missed the support thing
I also forgot to add a link, I'll do the when  in my PC